docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -30,6 +30,30 @@ description: >
 {{% alert title="General Information" color="info" %}}
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
+
+## [v0.19.10](https://github.com/aws/eks-anywhere/releases/tag/v0.19.10)
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.22.0 |    ✔    |     \*      |    —    |     —      |  —   |
```

Comment:
Do you happen to know why we removed the reference for what `*` means like we had in [v0.19.7](https://github.com/aws/eks-anywhere/releases/tag/v0.19.7) release?

Reply:
Good catch, added it for `0.19.9` and `0.19.10`.